tests/testthat/test-fill.R

```diff
@@ -0,0 +1,35 @@
+test_that('gh_fill works', {
+
+  # Test single string
+  expect_equal(gh_fill("9w", 3L), c("9w0", "9w1", "9w2", "9w3", "9w4", "9w5",
+                                    "9w6", "9w7", "9w8", "9w9", "9wb", "9wc",
+                                    "9wd", "9we", "9wf", "9wg", "9wh", "9wj",
+                                    "9wk", "9wm", "9wn", "9wp", "9wq", "9wr",
+                                    "9ws", "9wt", "9wu", "9wv", "9ww", "9wx",
+                                    "9wy", "9wz"))
+  # Test vector of inputs
+  expect_equal(gh_fill(c("9w", "9x"), 3L),
```

Comment:
I might expect the output here to be

```
list(
  c("9w0", ..., "9wz"),
  c("9x0", ..., "9xz")
)
```

That would make it much more natural to map inputs to outputs:

`fill( x[j] ) == out[[jj]]` instead of currently `fill( x[j] ) == out[1:32 + (32 * (jj - 1L))]`

WDYT?

We might also offer a `simplify=` (naming?) argument to toggle `unlist()` the version I propose.